internal/mode/static/nginx/config/main_config_template.go

```diff
@@ -21,9 +21,7 @@ mgmt {
 	resolver {{ .Resolver }};
 	{{- end }}
 	license_token {{ .LicenseTokenFile }};
-	{{- if .DeploymentCtxFile }}
-	deployment_context {{ .DeploymentCtxFile }};
-	{{- end }}
+	deployment_context /etc/nginx/main-includes/deployment_ctx.json;
```

Comment:
I think if we still need this conditional here. If there's any error that would result in the context file not having all required data (like failure to collect cluster information), then we'll be sending an invalid file which returns a 404 from N1 and stops traffic.